components/icons/images/tier2/classes.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+  <g fill="none" fill-rule="evenodd" stroke="#494C4E" stroke-width="2">
```

Comment:
@krshnpatel did you try this new version out on the demo page? I'm worried that the use of "stroke" instead of "fill" isn't going to work with the demo where it changes the colour.

Reply:
Ahhh yes, I did try it out on the demo page but missed that the colour actually didn't change for this logo 😓

Reply:
A quick fix you could try would be to use `fill` of `#494C4E` on the individual `<rect>`s.

Reply:
@geurts gave me another SVG that works perfectly on the demo page (i.e. colour override, size override) 😄